app/src/main/java/org/wikipedia/dataclient/mwapi/MwPostResponse.kt

```diff
@@ -0,0 +1,10 @@
+package org.wikipedia.dataclient.mwapi
+
+open class MwPostResponse : MwResponse() {
+    val pageInfo: MwQueryPage? = null
+    val options: String? = null
+    val successVal = 0
```

Comment:
You must never change the names of any fields in model classes, since these are deserialized from our JSON API, and must therefore match the server's response.